build.sh

```diff
@@ -1,7 +1,7 @@
 # This script is used for building a themed site to preview on render.com
 # Preview URL: https://pmm-doc.onrender.com
 
-python -m pip install --upgrade pip
-pip install wheel
+# python -m pip install --upgrade pip
```

Comment:
@rasika-chivate is this somehow related to the change?

Reply:
Actually not. But for successful deployment, this is required else all the builds are failing.